src/types/api/response.ts

```diff
@@ -17,7 +17,7 @@ export type GetMeResponse = {
   user: User
 }
 
-export type CreatePostResponse = {
+export type CreatedPostResponse = {
```

Comment:
```suggestion
export type CreatePostResponse = {
```

Because this is response of `create post request`